src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -563,11 +589,58 @@ public void ProcessCommand(IExecutionContext context, string inputLine, ActionCo
             context.AddIssue(issue);
         }
 
+        public static void ValidateLinesAndColumns(ActionCommand command) 
+        {
+            command.Properties.TryGetValue(IssueCommandProperties.Line, out string line);
+            command.Properties.TryGetValue(IssueCommandProperties.EndLine, out string endLine);
+            command.Properties.TryGetValue(IssueCommandProperties.Column, out string column);
+            command.Properties.TryGetValue(IssueCommandProperties.EndColumn, out string endColumn);
+
+            var hasStartLine = int.TryParse(line, out int lineNumber);
+            var hasEndLine = int.TryParse(endLine, out int endLineNumber);
+            var hasStartColumn = int.TryParse(column, out int columnNumber);
+            var hasEndColumn = int.TryParse(endColumn, out int endColumnNumber);
+            var hasColumn = hasStartColumn || hasEndColumn;
+
+            if (hasEndLine && !hasStartLine)
+            {
+                throw new Exception($"Invalid {command.Command} command value. 'end_line' can only be set if 'line' is provided");
```

Comment:
should we throw or just debug-log and ignore the value?

Reply:
personally wouldnt throw

Reply:
`ArgumentException` might be better than `exception`

Reply:
+1, what happens when we throw?
will the job fail?
the bottom line is I don't think a misconstructed logging command should fail the job.

Reply:
yep that's good feedback. I will change to just debug log and not fail